File: Java/URI 1038.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    
    int cod,qtd;
    double preço;
    cod = sc.nextInt();
    qtd = sc.nextInt();
    
    if(cod == 1){
        preço = qtd * 4.00;
    }else if(cod == 2){
        preço = qtd * 4.50;
    }else if(cod == 3){
        preço = qtd * 5.00;
    }else if(cod == 4){
        preço = qtd * 2.00;
    }else{
        preço = qtd * 1.50;
    }

    System.out.printf("Total: R$ %.2f%n",preço);
    sc.close();  
  }
}

File: C/URI 1038.c
#include <stdio.h>
int main(void) {

    int a,b; //a-codigo, b-qtd
    double total;
    scanf("%d %d",&a,&b);

    if(a == 1){
        total = b * 4.00;
        printf("Total: R$ %.2lf\n",total);
    }else if(a == 2){
        total = b * 4.50;
        printf("Total: R$ %.2lf\n",total);
    }else if(a == 3){
        total = b * 5.00;
        printf("Total: R$ %.2lf\n",total);
    }else if(a == 4){
        total = b * 2.00;
        printf("Total: R$ %.2lf\n",total);
    }else{
        total = b * 1.50;
        printf("Total: R$ %.2lf\n",total);
    }
    return 0;
}